linode/manager
# refactor: [M3-6273] - MUI v5 Migration - SRC > Features > Lish

## Description 📝
Refactored `Lish` as part of MUI v5 migration

## Changes  🔄
- `CircleProgress.tsx` - Allow `sx` to be passed in as props
- `ReachTab.tsx` - Remove `!important` that was causing styling issues with `Lish` tabs
- `Lish.tsx` - No need to import css files in `useEffect`, we can import normally.
   - Vertically center the `CircleProgress` when loading
   - Remove use of `Fragment` and return null instead
   - Use `Styled` api instead of `makeStyles`

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-10-10 at 10 21 12 AM](https://github.com/linode/manager/assets/125309814/d6d4d2e8-1dea-42e0-b97a-c61f0b4862e6) | ![Screenshot 2023-10-10 at 10 23 14 AM](https://github.com/linode/manager/assets/125309814/73bc56a9-dcb6-4826-a758-49a228797a40) |
|![Screenshot 2023-10-10 at 10 27 34 AM](https://github.com/linode/manager/assets/125309814/8f06ab87-8858-4d76-b636-d518d8cfb98e)|![Screenshot 2023-10-10 at 10 29 30 AM](https://github.com/linode/manager/assets/125309814/d959ed71-ff4d-4118-8c2c-9c2613a08f1c)|

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/125309814/87c01a35-d4e5-4a54-9264-780fab1a9a67" /> | <video src="https://github.com/linode/manager/assets/125309814/ec1c4b17-b14d-4fe1-86b0-29f760508dde" /> |
| <video src="https://github.com/linode/manager/assets/125309814/48c1b0af-0ad3-4dd9-b00a-f32bdb3b59ba" /> | <video src="https://github.com/linode/manager/assets/125309814/53df28da-5364-4216-b69c-bf1ea22d18d2" /> |

## How to test 🧪

### Prerequisites
- Must have a Linode

### Reproduction steps
- Go to linodes detail page
- Open "Launch LISH Console"

### Verification steps 
- Observe hover states for tabs are fixed
- Observe css styles are still working
- Observe loading state is vertically centered (alternatively if you're pulling this down locally, you can just set the `isLoading` condition to `true` to test

## As an Author I have considered 🤔

*Check all that apply*
